### DISCUSSION TITLE
CHeal chain timer

### DISCUSSION THREAD

**Discussion by Kritoris** [smasherprog/EqTool#118] [Ideas]
The CHeal chain is awesome, makes it very easy to see a gap and know when it is your turn.

It would be cool if each CHeal also had some kind of a timer between them so you can see the gap between each that is landing, and especially a count from when the last heal started so you know when to go depending on if it is a 2s chain, or an 8s chain for example. The tool would count up the seconds so you can cast exactly in time.

**Comment by smasherprog** [comment]
You can check these and receive Audio / Visual warnings when the person IN FRONT of you has gone. So you can be ready
<img width="289" alt="image" src="https://github.com/smasherprog/EqTool/assets/3393733/49861ba7-0e12-4cf6-ae02-ba6380f21fb3">

**Comment by smasherprog** [comment]
I like the timer which restarts for each CH so you can see the time from last cast easier. Ill work on that now
